src/Services/ActionManager.vala

```diff
@@ -152,6 +154,7 @@ public class Akira.Services.ActionManager : Object {
         typing_accelerators.set (ACTION_ELLIPSE_TOOL, "e");
         typing_accelerators.set (ACTION_TEXT_TOOL, "t");
         typing_accelerators.set (ACTION_IMAGE_TOOL, "i");
+        typing_accelerators.set (ACTION_PATH_TOOL, "p");
```

Comment:
Let's use `v` as accelerator since we're calling it the vector tool.
`p` will be used for the pencil too.